pkgs/applications/misc/wallgen/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonApplication, fetchFromGitHub, pillow, click, cython, scipy, flask, gevent, numpy, scikitimage, loguru }:
+
+buildPythonApplication rec {
+
+  pname = "wallgen";
```

For consistency's sake, the whitespace here should be removed.
```suggestion
buildPythonApplication rec {
  pname = "wallgen";
```